src/js/providers/utils/network-error-parser.js

```diff
@@ -12,7 +12,8 @@ export default function parseNetworkError(baseCode, statusCode, url) {
         code += clampStatus(statusCode);
     } else if (('' + url).substring(0, 5) === 'http:' && document.location.protocol === 'https:') {
         code += 12;
-    } else if (statusCode === 0) {
+    } else if (!statusCode) {
```

Comment:
We always expected a number here. Maybe we should handle this higher up the chain - like in HLS.js for example.